test/quantization/test_quantized_op.py

```diff
@@ -118,68 +118,172 @@ def _get_random_tensor_and_q_params(shapes, rand_scale, torch_type):
 
 class TestQuantizedOps(TestCase):
 
+    # The activation_test_config contains metadata about the quantized activation
+    # functions that will be tested. It allows simpler and more concise unit tests
+    # to be written by specifying the configurations needed for the test in the
+    # activation_test_config and call the provided helper function _test_activation.
+    # Here's a list of the fields that can be included in the config:
+    # quantized_fn: a list of the quantized functions to be tested
+    # reference_fn: the original reference function to be called on the
+    # the dequantized X
+    # inplace_kwarg: the additional inplace keyword argument to test in-place
+    # for each test entry in ops_under_test, it must have at least the fields
+    # for quantized_fn and reference_fn. If inplace_kwarg is missing, the
+    # quantized function is assumed to be either inplace by default or the
+    # test is not testing an inplace function.
+    # output_range: the output range the operator will map to. By default, if it is
+    # no specified, the range will be (-inf, inf)
+    activation_test_config = {
+        'relu': [
+            {
+                'quantized_fn': [
+                    torch.relu
+                ],
+                'reference_fn': torch.relu
+            },
+            {
+                'quantized_fn': [
+                    torch.relu_
+                ],
+                'reference_fn': torch.relu_,
+            },
+            {
+                'quantized_fn': [
+                    torch.nn.functional.relu,
+                    torch.nn.quantized.functional.relu,
+                ],
+                'reference_fn': torch.nn.functional.relu,
+            },
+            {
+                'quantized_fn': [
+                    torch.nn.functional.relu,
+                    torch.nn.quantized.functional.relu,
+                ],
+                'reference_fn': torch.nn.functional.relu,
+                'inplace_kwarg': {
+                    'inplace': True
+                }
+            }
+        ],
+        'relu6': [
+            {
+                'quantized_fn': [
+                    torch.ops.quantized.relu6
+                ],
+                'reference_fn': torch.nn.functional.relu6
+            },
+            {
+                'quantized_fn': [
+                    torch.nn.quantized.ReLU6()
+                ],
+                'reference_fn': torch.nn.ReLU6(),
+            },
+            {
+                'quantized_fn': [
+                    torch.nn.quantized.ReLU6(inplace=True)
+                ],
+                'reference_fn': torch.nn.ReLU6(),
+            }
```

Comment:
nit: It looks to me this configurations are very similar. Can you collapse them into one, and just add all of them to the list of `quantized_fn`?

```python
           {
                'quantized_fn': [
                    torch.ops.quantized.relu6,
                    torch.nn.quantized.ReLU6(inplace=False),
                    torch.nn.quantized.ReLU6(inplace=True)
                ],
                'reference_fn': torch.nn.functional.relu6
            },
```